addons/hr/models/hr_employee.py

```diff
@@ -499,6 +501,19 @@ def _launch_plan(self, plan_ids):
                     body='<br/>'.join(errors),
                 )
 
+    def _get_unusual_days(self, date_from, date_to=None):
+        # Checking the calendar directly allows to not grey out the leaves taken
```

Comment:
Hello @tivisse ,
Do you mind if we move the def under employee model?
It comes from the leave, but is compute from the employee, so it makes sense to me.

Thanks 🙂

Reply:
Hello @mba-odoo ,
I am wondering why this def has not been moved into hr.employee.base, as the resource_calendar_id field is actually added there.
What do you think?
After your return on this I will send the PR to YTI for review.
Thanks 🙂

Reply:
@IT-Ideas ,
We are calling this method with `user.employee_id._get_unusual_days` and `employee_id` field in user is in relation with the `hr.employee` not employee.base so make this method in hr.employee.

We may move this method in employee.base but employee.base is also inherited in employee.public and it's view so i don't think it's going to be more useful over there.

Let me know even if we wants to change it.

Thanks :)

Thanks